components/simplifiedGrid/simplifiedGrid.html

```diff
@@ -0,0 +1,298 @@
+              ></path>
+            </g>
+          </svg>
+        </nav>
+      </header>
+
+      <div class="main-body">
+        <aside class="main-sidebar">
+          <span class="sidebar-header">Components</span>
+          <ul class="sidebar-items">
+            <a href="../alert/alert.html" class="sidebar-item">Alert</a>
+            <a href="../avatar/avatar.html" class="sidebar-item">Avatar</a>
+            <a href="../badge/badge.html" class="sidebar-item">Badge</a>
+            <a href="../button/button.html" class="sidebar-item">Button</a>
+            <a href="../card/card.html" class="sidebar-item">Card</a>
+            <a href="../image/image.html" class="sidebar-item">Image</a>
+            <a href="../input/input.html" class="sidebar-item">Input</a>
+            <a href="../modal/modal.html" class="sidebar-item">Modal</a>
+            <a href="#" class="sidebar-item active">Simplified Grid</a>
+            <a href="../toast/toast.html" class="sidebar-item">Toast</a>
+          </ul>
+        </aside>
+        <div class="main-content-container">
+          <div class="main-content">
+            <h1 class="component-title">Grid</h1>
+            <p class="component-info">
+              A modal is a window laid over the main window to attract full
+              attention of the user. It can be dismissed by either clicking on
+              the buttons or the backdrop
+            </p>
```

Comment:
Here it has a modal description, I guess you've not changed the description here.

Reply:
Thank you, I have changed it.